### PR TITLE
Restructure and extend README file

### PR DESCRIPTION
**Pull request by sinarueeger**
## Description

Restructure and extend README file:
- added purpose
- added link to vignette
- restructured `usage` slightly (cosmetics)
- (side effect: added NCBI source to general package documentation (`man/rsnps-package.Rd `)

## Related Issue

#63

### CODE REVIEW

File: README.Rmd [hunk 1]
```diff
@@ -38,7 +40,7 @@ This set of functions/package accesses data from:
 	+ Relavant functions:
```
Comment (jooolia) [inline README.Rmd]: This PR looks very nice. Could you also fix this typo of "Relavant" to "Relevant" here?

Comment (sinarueeger) [reply README.Rmd]: Thanks for the pointer! Done. 
Also added this snippet to the `README.Rmd` file: 
```
---
output: github_document
---
<!-- README.md is generated from README.Rmd. Please edit that file -->

```
This way the Rmd file is rendered automatically to a md. file

**Comment by jooolia** [comment]
Hi @sinarueeger great it is merged. Are you ok with deleting branches after they are merged? Thanks, Julia

**Comment by sinarueeger** [comment]
Hi @jooolia - I always delete branches too!

**Pull request merged**